src/classes/BDI_CustObjMappingGAUAllocation.cls

```diff
@@ -76,15 +78,19 @@ public with sharing class BDI_CustObjMappingGAUAllocation extends BDI_ObjectMapp
                             paymentIdValue = value;
                         } else if (fieldMapping.Target_Field_API_Name == OPPORTUNITY_FIELDNAME) {
                             opportunityIdValue = value;
+                        } else if (fieldMapping.Target_Field_API_Name == RECURRING_DONATION_FIELDNAME) {
+                            recurringDonationIdValue = value;
                         } else {
                             castAndCopyField(objWrap, sourceFieldName, sourceFieldDescribe, targetFieldName, targetFieldDescribe);
                         }
                     }
                 }
             }
-
+            System.debug('CustObjMappingGAUAllocation: ' + settings.Payment_Allocations_Enabled__c);
```

Comment:
do you want to leave this for  future testing purposes?

Reply:
@grivero I'll take it out